context/app/static/js/shared-styles/dropdowns/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+import MenuItem from '@material-ui/core/MenuItem';
+import CheckRoundedIcon from '@material-ui/icons/CheckRounded';
+
+const iconSize = '1.25rem';
+
+const FlexMenuItem = styled(MenuItem)`
+  display: flex;
+`;
+
+const CheckIcon = styled(CheckRoundedIcon)`
+  font-size: ${iconSize};
+  margin-right: ${(props) => props.theme.spacing(1)}px;
+`;
+
+const StyledSpan = styled.span`
+  ${(props) => !props.isSelected && `margin-left: calc(${iconSize} + ${props.theme.spacing(1)}px)`};
```

Comment:
This is getting a little hairy. Any chance there's a class or something you could use to move the logic into CSS?

Reply:
Hairy as in it's difficult to read/interpret?